spec/controllers/admin/comments_controller_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'spec_helper'
+
+describe Admin::CommentsController, type: :controller do
+
+  # It is necessary to use bang version of let to build roles before user
+  let(:conference) { create(:conference) }
+  let!(:first_user) { create(:user) }
+  let!(:organizer_role) { create(:role, name: 'organizer', resource: conference) }
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.